WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsDeactivateStripeAdminView.swift

```diff
@@ -0,0 +1,85 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsDeactivateStripeAdmin: View {
```

Comment:
nit: what do you think of naming this struct with `View` at the end? If I just read this name I'd not have guessed this is a view. I understand that our codebase has similar naming for SwiftUI views, just wanted to mention since I saw we're good with naming SwiftUI views as `*View` in p1651699927182139/1651641104.110959-slack-CGPNUU63E (at least from the number of 👍 )